PP_APP/server/server/settings.py

```diff
@@ -17,7 +17,8 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
 GOOGLE_APPLICATION_CREDENTIALS = 'google-vision.json'
-MASK_RCNN_API_IP = '34.82.43.176'
+# MASK_RCNN_API_IP = '34.82.43.176' # M-RCNN API CLOUD
+MASK_RCNN_API_IP = '0.0.0.0' # M-RCNN RUNNING LOCAL
```

Comment:
Are we going to need to change this when necessary?

Reply:
Yeah, if the cloud server is on, you can comment out the local and uncomment the cloud. Otherwise, you have to setup the API locally and use it.